azure-mgmt-cosmosdb/src/main/java/com/microsoft/azure/management/cosmosdb/implementation/CosmosDBAccountImpl.java

```diff
@@ -162,6 +173,52 @@ public CosmosDBAccountImpl withKind(DatabaseAccountKind kind) {
         return this;        
     }
 
+    @Override
+    public CosmosDBAccountImpl withKind(DatabaseAccountKind kind, Capability... capabilities) {
+        this.inner().withKind(kind);
+        this.inner().withCapabilities(Arrays.asList(capabilities));
+        return this;
+    }
+
+    @Override
+    public CosmosDBAccountImpl withDataModelSql() {
+        this.inner().withKind(DatabaseAccountKind.GLOBAL_DOCUMENT_DB);
+        return this;
```

Comment:
Are you sure the "defaultExperience" tags are not needed on these? If you create each API type in the portal, they all have a "defaultExperience" tag.

Reply:
The tags do not impact resource provisioning; if they do then it's a bug. While the tests I've added set a generic tag it's not required to create an CosmosDB account.